pkgs/development/python-modules/textblob/default.nix

```diff
@@ -21,15 +21,13 @@ buildPythonPackage rec {
 
   dependencies = [ nltk ];
 
-  doCheck = true;
```

Comment:
Please add comment explaining why tests are disabled. (apparently they require network access)

Reply:
Sorry for the delay, I have added the comment on it.
I have also updated to sub version replacing #361489